cover_agent/UnitTestGenerator.py

```diff
@@ -201,14 +201,18 @@ def build_prompt(self):
             failed_test_runs_value = ""
             try:
                 for failed_test in self.failed_test_runs:
-                    code = failed_test['code'].strip()
+                    failed_test_dict = failed_test['code']
```

Comment:
Suggested by PR Agent:
```suggestion
                    failed_test_dict = failed_test.get('code', {})
```

Reply:
done